src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -3,27 +3,26 @@
 
 #pragma once
 
-#include "../../engine/sparqlExpressions/AggregateExpression.h"
-#include "../../engine/sparqlExpressions/GroupConcatExpression.h"
-#include "../../engine/sparqlExpressions/LiteralExpression.h"
-#include "../../engine/sparqlExpressions/NaryExpression.h"
-#include "../../engine/sparqlExpressions/SparqlExpressionPimpl.h"
-// #include "../../engine/sparqlExpressions/RelationalExpression.h"
-#include "../../engine/sparqlExpressions/SampleExpression.h"
-#include "../../util/HashMap.h"
-#include "../../util/OverloadCallOperator.h"
-#include "../../util/StringUtils.h"
-#include "../../util/antlr/ANTLRErrorHandling.h"
-#include "../Alias.h"
-#include "../ParsedQuery.h"
-#include "../RdfEscaping.h"
-#include "../data/BlankNode.h"
-#include "../data/Iri.h"
-#include "../data/SolutionModifiers.h"
-#include "../data/Types.h"
-#include "../data/VarOrTerm.h"
-#include "antlr4-runtime.h"
-#include "generated/SparqlAutomaticVisitor.h"
+#include <antlr4-runtime.h>
+#include <engine/sparqlExpressions/AggregateExpression.h>
+#include <engine/sparqlExpressions/GroupConcatExpression.h>
+#include <engine/sparqlExpressions/LiteralExpression.h>
+#include <engine/sparqlExpressions/NaryExpression.h>
+#include <engine/sparqlExpressions/SampleExpression.h>
+#include <engine/sparqlExpressions/SparqlExpressionPimpl.h>
+#include <parser/Alias.h>
+#include <parser/ParsedQuery.h>
+#include <parser/RdfEscaping.h>
+#include <parser/data/BlankNode.h>
+#include <parser/data/Iri.h>
+#include <parser/data/SolutionModifiers.h>
+#include <parser/data/Types.h>
+#include <parser/data/VarOrTerm.h>
+#include <parser/sparqlParser/generated/SparqlAutomaticVisitor.h>
+#include <util/HashMap.h>
+#include <util/OverloadCallOperator.h>
+#include <util/StringUtils.h>
+#include <util/antlr/ANTLRErrorHandling.h>
```

Comment:
Same order of the headers as stated above. 

(Blank line after the antlr header should be enough here.